lbz/resource.py

```diff
@@ -44,10 +42,6 @@ def __init__(self, event: dict):
             is_base64_encoded=event.get("isBase64Encoded", False),
             query_params=event["multiValueQueryStringParameters"],
         )
-        if authorization := headers.get("Authorization"):
```

Comment:
Fantastic job with this refactor, big thanks.

Reply:
+1